M2/Macaulay2/packages/Macaulay2Doc/functions/canUseHilbertHint-doc.m2

```diff
@@ -0,0 +1,61 @@
+doc ///
+  Key
+    canUseHilbertHint
+    (canUseHilbertHint, Ring)
+    (canUseHilbertHint, Ideal)
+    (canUseHilbertHint, Module)
+    (canUseHilbertHint, Matrix)
+  Headline
+    whether certain Groebner computations can make use of the Hilbert function
```

Comment:
I suspect nobody will actually find this node. How about this: add the example of passing the `Hilbert` option to one of the [many](https://github.com/Macaulay2/M2/issues/1347) nodes about Grobner bases instead, or add a separate node for the option `[gb, Hilbert]` and link to it from the documentation of `gb`. I would imagine `canUseHilbertHint` doesn't need to be exported in that case either.